### DISCUSSION TITLE
Glosbe

### DISCUSSION THREAD

**Discussion by RastikC** [dialect-app/dialect#339] [Ideas]
_Hello,_

would it, please, be possible to add **Glosbe** as a translation provider? They offer a number of languages not found elsewhere, such as **Interlingua** and many (stating **6000**) more.

https://app.glosbe.com/all-languages
https://app.glosbe.com/help/api
https://glosbe.com

_THANK YOU in advance._

_Best!_

**Comment by rafaelmardojai** [comment]
Thanks for the suggestion, didn't know of its existence.  Reading the docs it looks like the API was shut down, but I guess we could implement it using "unofficial" methods like we do for Google and alike.

**Comment by RastikC** [comment]
**THANKS for your positive feedback.** I wish you succeed implementing it (somehow).

_Best!_
